scan.rb

```diff
@@ -11,13 +11,33 @@ def get_option(key)
   selected.split('=').last
 end
 
+def write_results(hostname, response, output) 
+  begin
+    puts " Start writing final results"
+    file = File.open("#{output}", "ab+")
+    file.write("Found: #{hostname} (#{response.code})\n") 
+    response.to_hash.each do |header, values|
+      file.write("  #{header}:")
+      values.each do |value|
+        file.write("\n    #{value}\n")
+      end
+    end
+  rescue IOError => e
+    #some error occur, dir not writable etc.
+    puts e;
+  ensure
+    file.close unless file.nil?
+  end
+end 
+
 ip_address = get_option('ip')
 host = get_option('host')
 ignore_http_codes = get_option('ignore-http-codes') || '404'
 ignore_content_length = get_option('ignore-content-length') || 0
 wordlist_file = get_option('wordlist') || 'wordlist'
 ssl = get_option('ssl') || 'off'
 port = get_option('port') || (ssl == 'on' ? 443 : 80)
+output = get_option('o') || 'output.txt'
```

Comment:
Since all options are words, let's name it to `output` for now and have `-o` be introduced if someone would ever introduce a short notation for the options.

Reply:
I set it as output it makes more scenes, no need the short version.

Reply:
done